### PR TITLE
44 46 UI Details

### PR DESCRIPTION
**Pull request by jeffbuonamici**
#44 #46

- Created controllers for each scene
- Created CSS files for each scene
- Implemented the necessary JavaFX elements to display the data.
- Created Engine and Sensor classes to hold data in a usable format for the front end.
- Created the onClick events to link between scenes.

### CODE REVIEW

**Review by PaulMicu — CHANGES_REQUESTED** [review]
please remove target files from pull request

File: src/main/java/com/cbms/app/Engine.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
+package com.cbms.app;
+
+public class Engine {
```
Comment (rdsaliba) [inline src/main/java/com/cbms/app/Engine.java]: We created another model for this called "Asset"

**Review by rdsaliba — COMMENTED** [review]
I believe you must switch your engine class to our asset class to match the mimosa standard. Let me know what you think.

**Comment by jeffbuonamici** [comment]
I believe I fixed your requests.
- I'm now using Asset and AssetAttribute objects instead of Engine and Sensor
- Deleted target folder files that were committed

Also, Paul helped me update the branch from the Backend Connections branch to avoid merge errors I believe.

**Review by rdsaliba — APPROVED** [review]
Ready to merge

**Pull request merged**